app/components/AccountCard/AccountCard.view.tsx

```diff
@@ -65,14 +68,25 @@ const AccountCard: FC<AccountCardProps> = ({
 
   const handleToggleClick = () => setIsQRCode(!isQRCode);
 
+  const handleDialogClose = (event, reason) => {
+    // see https://v4.mui.com/api/dialog/
+    if (!reason) {
+      setLoading(false);
+    }
```

Comment:
this part is unclear. Why do we want to keep `loading` unchanged if the user exits the dialog from `backdropClick ` or `escapeKeyDown` reasons?

Reply:
we want to block the user from interacting with the app while the account switch is happening